samples/bluetooth/mesh/prj_bbc_microbit.conf

```diff
@@ -1,5 +1,5 @@
 CONFIG_INIT_STACKS=y
-CONFIG_ISR_STACK_SIZE=768
+CONFIG_ISR_STACK_SIZE=512
```

Comment:
Is there any confirmation if mesh works on this platform after reducing stack size?

Reply:
Reverted it as it was not needed when RELAY was removed.